inc/ticket.class.php

```diff
@@ -5238,11 +5238,16 @@ static function showListForItem(CommonDBTM $item) {
             // you can only see your tickets
             if (!Session::haveRight(self::$rightname, self::READALL)) {
                $restrict .= " AND (`glpi_tickets`.`users_id_recipient` = '".Session::getLoginUserID()."'
-                                   OR (`glpi_tickets_users`.`tickets_id` = '".$item->getID()."'
+                                   OR (`glpi_tickets_users`.`tickets_id` = `glpi_tickets`.`id`
                                        AND `glpi_tickets_users`.`users_id`
                                             = '".Session::getLoginUserID()."')
                                    OR `glpi_groups_tickets`.`groups_id` IN (".implode(",",$_SESSION['glpigroups'])."))";
             }
+
+            if (Session::haveRight(self::$rightname, self::ASSIGN)) {
```

Comment:
I've dropped this part; since this does not seems to be required to get the issue fixed.

@johannsan I do not understand what you want to achieve with that; maybe could you open another ticket explaining your need? Thank you :)